SpatialGDK/Source/SpatialGDK/Private/Utils/EntityFactory.cpp

```diff
@@ -0,0 +1,351 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+ 
+#include "Utils/EntityFactory.h"
+ 
+#include "Engine.h"
```

Comment:
UE4 includes should be the last ones to include.

Reply:
Addressed in 45c78cb